README.md

```diff
@@ -42,6 +42,16 @@ const client = truework({
 });
 ```
 
+To set a timeout on requests, pass in the timeout value in milliseconds when configuring the client. If a request times
+out, it will abort with [got.TimeoutError](https://www.npmjs.com/package/got#gottimeouterror).
```

Comment:
It doesn't feel great to leak this error, but we're already leaking `got` with the response types, so I think this is fine.

Reply:
Idk, I think our aim is to be a thin wrapper around a successful pattern in `got`. Re-implementing existing interfaces adds maintenance burden and potential for accidental for divergence.

Reply:
Maybe we can plug the sync handling here? Like "most helpful when used in conjunction with our [sync strategy]()". Only if you think it's helpful to readers.

Reply:
Yeah that's probably a good plug. And you're right, re-implementing would be a pain. I guess as long as we stick with got (and I don't know why we wouldn't) it's a non-issue.